docs/nodes/validate/add-a-validator.md

```diff
@@ -48,182 +48,308 @@ The response has your node’s ID:
 
-You should see this success message, and your balance should be updated.
+Calling [`platform.getPendingValidators`](../../apis/avalanchego/apis/p-chain.md#platformgetpendingvalidators) verifies that our transaction was accepted.
 
-![Your validation transaction is sent](/img/your-validation-transaction-is-sent.png)
+Go back to the `Earn` tab, and click `Estimated Rewards`.
 
-Calling [`platform.getPendingValidators`](../../apis/avalanchego/apis/p-chain.md#platformgetpendingvalidators) verifies that our transaction was accepted.
+Once your validator’s start time has passed, you will see the rewards it may earn, as well as its start time, end time, and the percentage of its validation period that has passed.
 
-![getPendingValidators postman](/img/getPendingValidators-postman.png)
+That’s it!
 
-Go back to the `Earn` tab, and click `Estimated Rewards`.
+## Add a validator with AvalancheJS
 
-![Earn, validate, delegate](/img/earn-validate-delegate.png)
+We can also add a node to the validator set using [AvalancheJS](../../apis/avalanchejs/README.md).
 
-Once your validator’s start time has passed, you will see the rewards it may earn, as well as its start time, end time, and the percentage of its validation period that has passed.
+### Install AvalancheJS
 
-![Estimated rewards](/img/estimated-rewards.png)
+To use AvalancheJS, you can clone the repo:
 
-That’s it!
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
+```
+
+For this tutorial we will use [`ts-node`](https://www.npmjs.com/package/ts-node) to run the example scripts directly from an AvalancheJS directory.
 
-## Add a validator with API calls
+### Fuji Workflow
 
-We can also add a node to the validator set by making API calls to our node. To add a node the Primary Network, we’ll call [`platform.addValidator`](../../apis/avalanchego/apis/p-chain.md#platformaddvalidator).
+Open your AvalancheJS directory and select the [**`examples/platformvm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/platformvm) folder to view the source code for the examples scripts.
 
-This method’s signature is:
+We will use the [**`buildAddValidatorTx.ts`**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildAddValidatorTx.ts) script to add a validator.
 
+#### Private Key
+
+Locate this line in the file
+
+```js
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
 ```
-platform.addValidator(
-    {
-        nodeID: string,
-        startTime: int,
-        endTime: int,
-        stakeAmount: int,
-        rewardAddress: string,
-        changeAddr: string, (optional)
-        delegationFeeRate: float,
-        username: string,
-        password: string
-    }
-) -> {txID: string}
+
+and replace this with a private key that you control.
+
+```js
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
```

Comment:
You should add a node about trying to use a Ledger WHENEVER you can.

Reply:
Definitely!

I was unaware that we can use a ledger to do this. 
Is [this](https://docs.avax.network/apis/avalanche-wallet-sdk/wallet-classes/#ledger-wallet) something I can use to illustrate using a ledger?

Reply:
This will be addressed by https://ava-labs.atlassian.net/browse/DS-208